sycl/test/regression/usm_malloc_shared.cpp

```diff
@@ -0,0 +1,50 @@
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: env SYCL_DEVICE_TYPE=CPU %t.out
+// RUN: env SYCL_DEVICE_TYPE=GPU %t.out
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+
+//==-------------- usm_free.cpp - SYCL USM free malloc_shared and free test
+//-------------==//
+//
+// This test checks if users will successfully allocate 160, 0, and -16 bytes of
+// shared memory, and also test user can call free() without worrying about
+// nullptr or invalid memory descriptor returned from malloc.
+//==-------------------------------------------------------------------------------------==//
+
```

Comment:
```suggestion

```

Reply:
@bader, Usually it is better to have some comments about the purpose of this test case. Further, many other regression tests already have this comment section.
Do you think this is not very useful?

Reply:
I meant to suggest removing lines 7, 8, 9 and 13, which value I don't understand and formatting of 7 and 8 seem to be broken.

Also, IMHO, the test case is trivial enough to understand what is checked, but I don't insist on removing 10-12.

Reply:
I just copied the existing comment format from same_unnamed_kernels.cpp in the same directory.
Okay, I will remove line 7-9 and 13, keeping lines 10-12. 
I will also run Clang-format again to see if the format follows the guideline.